examples/matrix_and_task_flow.yaml

```diff
@@ -0,0 +1,13 @@
+options:
+  log_dir: example_logs
+  limit: 1
+dependencies:
+- package: openai
+- package: git+https://github.com/UKGovernmentBEIS/inspect_evals@dac86bcfdc090f78ce38160cef5d5febf0fb3670
+matrix:
+  tasks:
+  - name: inspect_evals/mmlu_0_shot
+    models:
+    - name: openai/gpt-4o-mini
+    - name: openai/gpt-5-nano
+  - name: inspect_evals/mmlu_5_shot
```

Comment:
What model would `inspect_evals/mmlu_5_shot` get evaluated with? I imagine it would throw an error (unless `INSPECT_EVAL_MODEL` is set or the model is hardcoded in the task) while `inspect_evals/mmlu_0_shot` would run with the two models specified?